packages/zwave-js/src/lib/telemetry/metrics.ts

```diff
@@ -0,0 +1,90 @@
+import { formatId } from "@zwave-js/shared";
+import { isObject } from "alcalzone-shared/typeguards";
+import axios from "axios";
+import * as crypto from "crypto";
+import type { Driver } from "../driver/Driver";
+
+const baseUrl = "https://metrics.zwave-js.io:9998"; // No leading slash!
+// The shared "secret" between metrics server and zwave-js. Can be used to disable reporting for misbehaving clients.
+const authKey = "Og9O}$[wKk=<$Z*qUduK";
+
+const authUrl = `${baseUrl}/auth`;
+const metricsUrl = `${baseUrl}/metrics`;
+
+export interface AppInfo {
+	driverVersion: string;
+	applicationName: string;
+	applicationVersion: string;
+}
+
+export function compileMetrics(
+	driver: Driver,
+	appInfo: AppInfo,
+): Record<string, any> {
+	return {
+		// Hash the homeId, so it cannot easily be tracked
+		id: crypto
+			.createHash("md5")
```

Comment:
why md5, not sha (SHA-256)?

Reply:
Doesn't really matter, we can do sha. The thought was: MD5 is shorter 🤷🏻‍♂️